internal/ansible/controller/reconcile.go

```diff
@@ -63,7 +63,7 @@ type AnsibleOperatorReconciler struct {
 }
 
 // Reconcile - handle the event.
-func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) { //nolint:gocyclo
```

Comment:
IHMO here we need to add TODO: Try to reduce the complexity of this and remove the // nolint:gocyclo.